Intelipro/reja
# Feat: Minor app changes

#### What does this PR do?
 Implements minor changes on the application

#### Description of Task to be completed?
- [x] Add a close icon on modals
- [x] Clear selected fields on selecting different upload file
- [x] Filter out selected dropdown fields on the upload section on specific tables step
- [x] Make 365 days the default value on the dashboard page
- [x] Remove The word `Total` from the header button on the dashboard page
- [x] Add percentage sign for the revenue growth button on revenue page
- [x] Rename average transaction button to average revenue on revenue page

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `ft-app-minor-changes`
- Install all dependencies `npm install`
- To test the changes, confirm against the tasks completed above 

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)

#### Questions:
